modules/shell/example_test.go

```diff
@@ -0,0 +1,35 @@
+package shell_test
+
+import (
+	"testing"
+
+	"github.com/gruntwork-io/terratest/modules/shell"
+)
+
+func ExampleCommand() {
```

Comment:
`t.Parallel()`

Reply:
Oh, just realized this isn't a valid test function...

Reply:
It should be named `TestXXX` and take a `t *testing.T` as a param.

Reply:
It's not a test, it's an example:
https://blog.golang.org/examples